packages/features/ee/workflows/lib/reminders/emailReminderManager.ts

```diff
@@ -105,12 +105,13 @@ export const scheduleEmailReminder = async (
   sendTo: MailData["to"],
   emailSubject: string,
   emailBody: string,
-  workflowStepId: number,
   template: WorkflowTemplates,
   sender: string,
+  workflowStepId?: number,
   hideBranding?: boolean,
   seatReferenceUid?: string,
-  includeCalendarEvent?: boolean
+  includeCalendarEvent?: boolean,
+  isMandatoryReminder?: boolean
```

Comment:
we should use object as parameter as they are too many parameters and some are even optional

Reply:
Agree, changed to object parameter for `scheduleEmailReminder`, `scheduleSMSReminder` and `ScheduleWhatsappReminder`